demos/iperf3/testdata/manifests/iperf3-client/iperf3-client.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       containers:
       - name: iperf3-client
         #image: networkstatic/iperf3
-        image: mlabbe/iperf3
+        image: taoyou/iperf3-alpine
```

Comment:
lighter iperf image, I suppose? Or does it support multi-arch which `mlabbe` does not?

Reply:
Yes, It also compiles to arm64